CMakeLists.txt

```diff
@@ -104,9 +104,12 @@ if(WIN32)
     add_definitions(-D_WIN32_WINNT=0x0602)
 endif()
 
-if(APPLE)
-    include_directories(/opt/local/include)
-    link_directories(/opt/local/lib)
+if(APPLE AND CMAKE_OSX_ARCHITECTURES STREQUAL "arm64")
+    include_directories(/opt/homebrew/include)
+    link_directories(/opt/homebrew/lib)
+elseif(APPLE AND CMAKE_OSX_ARCHITECTURES STREQUAL "x86_64")
```

Comment:
We don't need this path, we can just keep it as an arm64 check. `/usr/local` is part of the default search path.

Reply:
I just tested this and yes, there is no error on Intel if the x86 check is removed.